corehq/apps/fixtures/views.py

```diff
@@ -456,7 +456,7 @@ def fixture_api_upload_status(request, domain, download_id, **kwargs):
     except TaskFailedError as e:
         notify_exception(request, message=six.text_type(e))
         response = {
-            'message': _("Upload did not complete. Reason: '{}'".format(e.message)),
+            'message': _("Upload did not complete. Reason: '{}'".format(six.text_type(e))),
```

Comment:
nit: I think `six.text_type()` is not necessary here since it would be converted to a string (`e.__str__()` would be called) by `"... '{}'".format(e)`